0.10/s2i/bin/assemble

```diff
@@ -22,6 +22,9 @@ shopt -s dotglob
 echo "---> Installing application source ..."
 mv /tmp/src/* ./
 
+echo "--> Setting npm loglevel to ${NPM_LOG_LEVEL:-warn}"
+npm config set loglevel ${NPM_LOG_LEVEL:-warn}
```

Comment:
I know I suggested the above, but on second thought I think it is better to follow the pattern set by the other env vars.

```bash
 if [ ! -z $NPM_LOG_LEVEL ]; then
  	echo "---> Setting npm loglevel to $NPM_LOG_LEVEL"
  	npm config set loglevel $NPM_LOG_LEVEL
 fi
```

The advantage here is that if NPM_LOG_LEVEL is not set we do nothing, instead of changing the config. That's preferable because if there is any way to provide custom configuration (.npmrc?) we would not interfere with values set there.